Build.md

```diff
@@ -46,7 +46,7 @@
   - Run the following two commands
 
     ```
-    cmake .. -DCMAKE_BUILD_TYPE=Release
+    cmake .. -T v140_xp -DCMAKE_BUILD_TYPE=Release
```

Comment:
The cmake man page calls it the "Toolset specification for the generator, if supported". For the VS generator, it selects the "platform toolset", which AFAIK is Windows terminology for which C++ runtime the project will use. v140_xp is the VS2015 runtime, with compatibility down to win XP. We're already using `-T v140_xp` in the appveyor scripts, so this is making the build.md instructions consistent with that.